lib/cmds/users-cmds/delete.js

```diff
@@ -0,0 +1,22 @@
+const csv = require('./../../util/csv');
+const config = require('./../../util/config');
+const intercom = require('./../../util/intercom')(config);
+const log = require("./../../util/log");
+
+const deleteUsers = (file, dir = `${__dirname}/../../../out`) => new Promise(resolve => csv
+  .read(`${dir}/${file}`)
+  .then((dataList) => {
+    let count = 0;
+    return intercom.users.bulk((data) => {
+      count += 1;
+      if (count % 20 === 1) {
+        log.info(`Processing ${count} / ${dataList.length}`);
+      }
+      return intercom.users.delete(data);
+    }, dataList);
+  }).then(resolve));
```

Comment:
I don't think we need the outer Promise resolve wrapper? Where else is this redundant?

Reply:
It never finishes the Promise without it though, I'm not sure this is redundant